src/Network/GuzzleResponse.php

```diff
@@ -88,7 +88,7 @@ public function getReturnCode()
 	/** {@inheritDoc} */
 	public function getContentType()
 	{
-		return $this->getHeader('Content-Type');
+		return implode($this->getHeader('Content-Type'));
```

Comment:
Don't you need to mention the glue character?

Reply:
Additionally, if there can only be one Content-Type, then if there are multiple values defined in the header, wouldn't the last one be used instead of a combination of them?

Reply:
> Don't you need to mention the glue character?

https://www.php.net/manual/en/function.implode.php
No, the glue is optional and per default empty


> Additionally, if there can only be one Content-Type, then if there are multiple values defined in the header, wouldn't the last one be used instead of a combination of them?

More strict, there should be an exception in case there are two headers?

Reply:
I'd rather have the last one because it is more likely to be useful than a concatenation of them, especially without a glue character.